nanoFramework.System.Text.Benchmark/Program.cs

```diff
@@ -0,0 +1,22 @@
+using nanoFramework.Benchmark;
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace nanoFramework.System.Text.Benchmark
+{
+    public class Program
+    {
+        public static void Main()
+        {
+
+            Debug.WriteLine("Hello from nanoFramework JSON benchmark!");
```

Comment:
you'll then be able to adjust this one ;-)

Reply:
removed